app/helpers/enterprises_helper.rb

```diff
@@ -14,15 +14,7 @@ def current_customer
   end
 
   def available_shipping_methods
-    return [] if current_distributor.blank?
-
-    shipping_methods = current_distributor.shipping_methods.display_on_checkout.to_a
-
-    applicator = OpenFoodNetwork::TagRuleApplicator.new(current_distributor,
-                                                        "FilterShippingMethods", current_customer&.tag_list)
-    applicator.filter!(shipping_methods)
-
-    shipping_methods.uniq
+    OrderAvailableShippingMethods.new(current_order, current_customer).to_a
```

Comment:
Yes I might leave this one because I thought we could introduce an `OrderAvailablePaymentMethods` service for the next part of #8971, to let people customise which payment methods are available on order cycles.